extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -240,6 +240,8 @@ export async function getCompletionItemsFromSpecs(
 	let foldersRequested = false;
 
 	const precedingText = terminalContext.commandLine.slice(0, terminalContext.cursorPosition + 1);
+	// TODO: Normalize precedingText to ignore file extensions on Windows
+	// precedingText = precedingText.replace('.cmd', '');
```

Comment:
I left in these comments to help when I do https://github.com/microsoft/vscode/issues/237598 probably today